exasol/python_extension_common/deployment/language_container/flavor_base/language_definition

```diff
@@ -0,0 +1 @@
+PYTHON3_TE=localzmq+protobuf:///{{ bucketfs_name }}/{{ bucket_name }}/{{ path_in_bucket }}{{ release_name }}?lang=python#buckets/{{ bucketfs_name }}/{{ bucket_name }}/{{ path_in_bucket }}{{ release_name }}/exaudf/exaudfclient_py3
```

Comment:
I think, we should replace PYTHON3_TE with something specified by the user

Reply:
Oops, missed that. Thanks for pointing out.